package.json

```diff
@@ -388,7 +401,13 @@
 				{
 					"when": "hediet.vscode-drawio.active",
 					"command": "hediet.vscode-drawio.linkFileWithSelectedNode",
-					"group": "Drawio"
+					"group": "Draw.io"
+				}
+			],
+			"commandPalette": [
+				{
+					"when": "false",
+					"command": "hediet.vscode-drawio.linkFileWithSelectedNode"
```

Comment:
Why removing this command from the command palette?

Reply:
Invoking this from the palette will fail as it is expecting an extra argument only present when calling it from the context menu. Removed to avoid accidentally using it from the palette.

Reply:
Ah, makes sense, thanks!